packages/@webex/internal-plugin-device/test/unit/spec/device.js

```diff
@@ -273,6 +273,17 @@ describe('plugin-device', () => {
         assert.notCalled(webex.internal.newMetrics.submitInternalEvent);
       });
 
+      it('sets the deviceInfo for call diagnostic metrics', async () => {
+        setup();
+        sinon.stub(device, 'canRegister').callsFake(() => Promise.resolve());
+        sinon.spy(device, 'request');
+
+        await device.register();
+
+        assert.called(webex.internal.newMetrics.callDiagnosticMetrics.setDeviceInfo);
```

Comment:
nitpick: would be good to assert that it's called with something

Reply:
Updated it